unit-tests/dds/d405.py

```diff
@@ -204,67 +204,62 @@ def color_stream():
+    option = dds.option.from_json( ["Hue", 0, -180, 180, 1, 0, "UVC image hue"] )
     options.append( option )
-    option = dds.option( "Saturation", dds.option_range( 0, 100, 1, 64 ), "UVC image saturation setting" )
+    option = dds.option.from_json( ["Saturation", 64, 0, 100, 1, 64, "UVC image saturation setting"] )
     options.append( option )
-    option = dds.option( "Sharpness", dds.option_range( 0, 100, 1, 50 ), "UVC image sharpness setting" )
+    option = dds.option.from_json( ["Sharpness", 50, 0, 100, 1, 50, "UVC image sharpness setting"] )
     options.append( option )
-    option = dds.option( "White Balance", dds.option_range( 2800, 6500, 10, 4600 ),
-        "Controls white balance of color image. Setting any value will disable auto white balance" )
+    option = dds.option.from_json( ["White Balance", 4600, 2800, 6500, 10, 4600,
+        "Controls white balance of color image. Setting any value will disable auto white balance"] )
     options.append( option )
-    option = dds.option( "Enable Auto Exposure", dds.option_range( 0, 1, 1, 1 ), "Enable Auto Exposure" )
+    option = dds.option.from_json( ["Enable Auto Exposure", 1, 0, 1, 1, 1, "Enable Auto Exposure"] )
     options.append( option )
-    option = dds.option( "Enable Auto White Balance", dds.option_range( 0, 1, 1, 1 ), "Enable / disable auto-white-balance" )
+    option = dds.option.from_json( ["Enable Auto White Balance", 1, 0, 1, 1, 1, "Enable / disable auto-white-balance"] )
     options.append( option )
-    option = dds.option( "Visual Preset", dds.option_range( 0, 5, 1, 0 ), "Advanced-Mode Preset" )
+    option = dds.option.from_json( ["Visual Preset", 0, 0, 5, 1, 0, "Advanced-Mode Preset"] )
     options.append( option )
-    option = dds.option( "Power Line Frequency", dds.option_range( 0, 3, 1, 3 ), "Power Line Frequency" )
+    option = dds.option.from_json( ["Power Line Frequency", 3, 0, 3, 1, 3, "Power Line Frequency"] )
     options.append( option )
-    option = dds.option( "Error Polling Enabled", dds.option_range( 0, 1, 1, 0 ), "Enable / disable polling of camera internal errors" )
-    option.set_value( 1 )
+    option = dds.option.from_json( ["Error Polling Enabled", 1, 0, 1, 1, 0, "Enable / disable polling of camera internal errors"] )
     options.append( option )
-    option = dds.option( "Output Trigger Enabled", dds.option_range( 0, 1, 1, 0 ), "Generate trigger from the camera to external device once per frame" )
+    option = dds.option.from_json( ["Output Trigger Enabled", 0, 0, 1, 1, 0, "Generate trigger from the camera to external device once per frame"] )
     options.append( option )
-    option = dds.option( "Depth Units", dds.option_range( 1e-06, 0.01, 1e-06, 0.001 ), "Number of meters represented by a single depth unit" )
-    option.set_value( 0.0001 )
+    option = dds.option.from_json( ["Depth Units", 0.0001, 1e-06, 0.01, 1e-06, 0.001, "Number of meters represented by a single depth unit"] )
```

Comment:
0.001

Reply:
I though you take the default. Now I see that other options also don't use default. So as long as it is in the range I guess it's OK

Reply:
Yeah, we now pass both value and default-value in the JSON, so no need to set_value after declaring.